### PR TITLE
README: clean up a bit

### PR DESCRIPTION
**Pull request by woodruffw**
Cleans up the README a bit and addresses some RuboCop lints.

Should be squashed during merge.

### CODE REVIEW

File: cmd/verify.rb [hunk 1]
```diff
@@ -35,7 +35,7 @@ def self.verify_args
              description: "Also download dependencies for any listed <formula>."
       switch "-f", "--force",
              description: "Remove a previously cached version and re-fetch."
-      conflicts "--formula", "--cask"
+      conflicts "--formula"
```
Comment (josephsweeney) [inline cmd/verify.rb]:
```suggestion
```

Comment (josephsweeney) [reply cmd/verify.rb]: I believe `conflicts` is just preventing the user from passing in two conflicting options, so it would need two args.

**Review by josephsweeney — CHANGES_REQUESTED** [review]
Thanks, I was thinking about whether or not we want to support casks as well which is why I had left those options in. I think we can remove for clarity now though and get everything working with just the formulae.

File: cmd/verify.rb (hunk 1 above)
Comment (woodruffw) [reply cmd/verify.rb]: Good catch, thanks!

**Comment by woodruffw** [comment]
> Thanks, I was thinking about whether or not we want to support casks as well which is why I had left those options in. I think we can remove for clarity now though and get everything working with just the formulae.

Yeah, this is the right path forwards IMO -- Cask signing is an entire can of worms, since it's not clear what signing communicates for third-party application builds.

**Review by josephsweeney — APPROVED** [review]

**Pull request merged**